src/model/text.js

```diff
@@ -33,7 +33,7 @@ export default class Text extends Node {
 		 *
 		 * @type {String}
 		 */
-		this.data = data || '';
+		this.data = data ? data.normalize() : '';
```

Comment:
It may break composition and it proved to be painful during tests. After all, I think we should not normalize this. The only problem will be for features that scan texts for given value (i.e. "search feature"). Because user might try to search for normalized version and there could be not-normalized version in text.

But I think it's better to not normalize.